### ISSUE TITLE
[BUG] belowaverage RULE triggers when should NOT

### ISSUE THREAD

**Issue by awedy01** [arkayenro/arkinventory#1538]
**What version number you are using?**
v3.09.60(should be latest, i copied this from /arkinventory config)

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Retail / Live

**What language is the game client set to?**
English

**Describe the bug**
Items will be JUNK when they arent.
The items will be counted as if this rule is true for them: "soulbound() and belowaverage( 20, 2 ) and not outfit() and equip()"

**To Reproduce**
Have the rule mentioned above, just play normally, so far what i was doing:
log off on a character while flying, AFK in the Maw, Die once, farm Elethium & Nightshade, kill a few mobs there, keep running from them, trade with friend(get item from him), HS to Heart of the Forest.

**Screenshots**
![image](https://user-images.githubusercontent.com/57992981/141647359-60d9ba8a-1b5e-4909-b556-f5e47f9490ca.png)

**Additional context**
THIS IS not the way to reproduce, i just dont know what actually caused it, so i wrote down the things i done in this session.
In short, this is caused by the new rule you done (for me?). The thing that sell item if its below my average item level. this bug gets fixed after reload, or even after removing the content of rule & writing "false" as if its a default rule, Then i write the content back to the rule and bug is fixed just as if i reload.

**Comment by arkayenro** [comment]
im not sure why it would do that (nor could i reproduce it) but i've rearranged that rule functions code where i thought it might be possible to happen

**Comment by awedy01** [comment]
I wait for an alpha release or so & ill keep an eye on it to see if it happens again!
